libs/wingsdk/src/shared-aws/function.inflight.ts

```diff
@@ -110,7 +113,8 @@ function parseCommandOutput(
   if (!payload.Payload) {
     return undefined;
   } else {
-    return JSON.parse(toUtf8(payload.Payload));
+    const returnObject = JSON.parse(toUtf8(payload.Payload));
+    return returnObject === null ? undefined : returnObject;
```

Comment:
This shouldn't be needed -- `==` in the language treats null and undefined as the same (I believe that's the semantics we want)

Reply:
Maybe a bug with expect.equal?

Reply:
I guess you're right that this is a bug with expect.equal though. Shouldn't we still avoid `null` bubbling through at runtime though?

Reply:
It's an open decision. Since Wing doesn't distinguish between null and undefined in its type system, I think there are two ways things can work when we emit and run JavaScript code:
1. enforce that only one of them is used (e.g. use "undefined" everywhere). This means if someone writes an extern function typed as returning `str?` and their JavaScript code implementation returns a null instead of undefined, then mysterious bugs may start occurring.
2. allow them to be used interchangeably. This puts a bit more burden on us (SDK and compiler internals need to be permissive to either appearing) but results in fewer errors for users, IMO. It's possible there are holes in this approach I'm not thinking of though.

In any case I'm cool with converting null's into undefined's by hand here as quick fix

Reply:
In the context of this specific function in TypeScript, this ternary is technically just doing a better job of ensuring we are returning the correct thing. The return type here is `string | undefined`, not `string | undefined | null`.